dandalandone/POC-EyCommerce
# [Feat] Product Detail API Implementation

## Ticket/s

[Product Info - Form](https://trello.com/c/rgTHWlSJ)

## Changes

- Implement Add to Cart API
- Add Quantity Button
- Add Product Inventory
- Modify Product Info Component Styling

## Screenshot

![image](https://github.com/TreshAnn/POC-EyCommerce/assets/126449551/5fbb5b60-4e59-4222-9191-ea7cf286f916)

## How to test?

Note: Ensure that you have user data and sample products available before testing.
- Login first
- Open the following URL in your browser: `http://localhost:5173/products`
- Click on any product card to navigate to the Product Detail View.
- Verify that the Product Detail View displays the correct details for the selected product.
- Modify Quantity and Click 'Add to Cart' Button
